### PR TITLE
Chinese Traditional

### PR DESCRIPTION
**Pull request by timoballard**
[Link to ZenHub issue.](https://app.zenhub.com/workspaces/doj-crt-intake-scrum-board-5d03bf56c1c8a35d482eca0f/issues/18f/crt-portal/206)

## What does this change?
Adds support for Traditional Chinese. Latest translation document can be referenced [here](https://drive.google.com/file/d/1jn_2Iq9T5tGlFc3mqTz3Y1Ddh37MVZHz/view?usp=sharing).

### Author

+ [ ] If this is a story, run locally and check to make sure all Acceptance Criteria are met. (If any criteria are unclear, ask about them.).
+ [x] Check for, document, and establish a testing plan for any behavior that may vary across environments or is otherwise difficult to test.
+ [x] Check for [accessibility](/docs/a11y_plan.md).
+ [x] [Tests pass](https://github.com/USDOJ/crt-portal/#tests).

### Reviewer

+ [ ] If this is a story, run locally and check to make sure all Acceptance Criteria are met. (If any criteria are unclear, ask about them.).
+ [ ] Check for any behavior that may vary across environments or is difficult to test, and ensure that it is well-understood, documented, and that there is a testing plan in place.
+ [ ] Re-check for [accessibility](/docs/a11y_plan.md).
+ [ ] [Tests pass](https://github.com/USDOJ/crt-portal/#tests).

## Notes for reviewer:

See [PR instructions doc](https://github.com/usdoj/crt-portal/blob/master/docs/pull_requests.md) for full pull request review instructions.

### CODE REVIEW

File: crt_portal/cts_forms/locale/zh_hant/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -0,0 +1,2491 @@
...
+msgstr "進度"
+
+#: templates/forms/portal/progress-bar.html:8
+#, python-format
+msgid ""
+"\n"
+"          Current step: %(current_step_name)s. Step %(stage_number)s of "
+"%(total_stages)s.\n"
+"        "
+msgstr ""
```
Comment (Jkrzy) [inline crt_portal/cts_forms/locale/zh_hant/LC_MESSAGES/django.po:1427]: Missing translation

File: crt_portal/cts_forms/locale/zh_hant/LC_MESSAGES/django.po [hunk 2]
```diff
@@ -0,0 +1,2491 @@
...
+#: templates/forms/portal/progress-bar.html:29
+msgid "completed"
+msgstr "已完成"
+
+#: templates/forms/report_base.html:8
+#, python-format
+msgid ""
+"\n"
+"        Step %(stage_number)s: %(current_step_name)s - "
+msgstr ""
```
Comment (Jkrzy) [inline crt_portal/cts_forms/locale/zh_hant/LC_MESSAGES/django.po:1442]: Missing translation

File: crt_portal/cts_forms/locale/zh_hant/LC_MESSAGES/django.po [hunk 3]
```diff
@@ -0,0 +1,2491 @@
...
+msgid "Examples:"
+msgstr "例子："
+
+#: templates/forms/word_counter.html:4
+#, python-format
+msgid ""
+"\n"
+"    %(word_limit)s word limit.\n"
+"  "
+msgstr ""
```
Comment (Jkrzy) [inline crt_portal/cts_forms/locale/zh_hant/LC_MESSAGES/django.po:1552]: Missing translation

File: crt_portal/cts_forms/locale/zh_hant/LC_MESSAGES/django.po [hunk 4]
```diff
@@ -0,0 +1,2491 @@
...
+
+#: templates/forms/word_counter.html:16 templates/forms/word_counter.html:21
+#, python-format
+msgid ""
+"\n"
+"          %(word_limit)s word limit reached\n"
+"        "
+msgstr ""
```
Comment (Jkrzy) [inline crt_portal/cts_forms/locale/zh_hant/LC_MESSAGES/django.po:1560]: Missing translation

File: crt_portal/cts_forms/locale/zh_hant/LC_MESSAGES/django.po [hunk 5]
```diff
@@ -0,0 +1,2491 @@
...
+msgid "Continue"
+msgstr "繼續"
+
+#: templates/partials/unsupported-browser-modal.html:10
+msgid ""
+"\n"
+"          <strong>NOTICE: Your mobile browser is not compatible with this "
+"form</strong> and you may experience issues with form submission.\n"
+"          "
+msgstr ""
```
Comment (Jkrzy) [inline crt_portal/cts_forms/locale/zh_hant/LC_MESSAGES/django.po:2069]: Missing translation

File: crt_portal/cts_forms/locale/zh_hant/LC_MESSAGES/django.po [hunk 6]
```diff
@@ -0,0 +1,2491 @@
...
+
+#: templates/partials/unsupported-browser-modal.html:15
+msgid ""
+"\n"
+"          Please consider downloading the latest version of Google Chrome, "
+"Mozilla Firefox, Safari, or Microsoft Edge. <a id=\"ub_contact\" href="
+"\"#contact-crt\">You can also contact us by phone or mail here.</a>\n"
+"          "
+msgstr ""
```
Comment (Jkrzy) [inline crt_portal/cts_forms/locale/zh_hant/LC_MESSAGES/django.po:2078]: Missing translation

File: crt_portal/cts_forms/locale/zh_hant/LC_MESSAGES/django.po [hunk 7]
```diff
@@ -0,0 +1,2491 @@
...
+msgstr "圖片來源"
+
+#: templates/partials/attachment-uploading-modal.html:8
+msgid "Uploading attachment..."
+msgstr ""
+
+#: templates/partials/attachment-uploading-modal.html:13
+msgid "Your attachment is uploading..."
+msgstr ""
+
```
Comment (Jkrzy) [inline crt_portal/cts_forms/locale/zh_hant/LC_MESSAGES/django.po:1910]: We should remove the `gettext` calls, so they're not included in `django.po` files, from these strings until we intend to translate them. At the moment these are only rendered on internal, English only, pages.

File: crt_portal/cts_forms/locale/zh_hant/LC_MESSAGES/django.po [hunk 8]
```diff
@@ -0,0 +1,2491 @@
+# This file was generated from /Users/timothydballard/temp/zh-hant.xlsx
```
Comment (Jkrzy) [inline crt_portal/cts_forms/locale/zh_hant/LC_MESSAGES/django.po:1]: Dev remnant

File: crt_portal/cts_forms/locale/zh_hant/LC_MESSAGES/django.po [hunk 9]
```diff
@@ -0,0 +1,2491 @@
...
+msgid ""
+msgstr ""
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2021-04-08 16:55+0000\n"
+"PO-Revision-Date: 2021-03-31 17:42-0500\n"
+"Language: Chinese Traditional\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Generated-By: xls-to-po 1.0\n"
+"Plural-Forms: nplurals=1; plural=0;\n"
```
Comment (Jkrzy) [inline crt_portal/cts_forms/locale/zh_hant/LC_MESSAGES/django.po:11]: Good opportunity to make these consistent across `.po `files now and set the pattern going forward for other languages.

Comment (timoballard) [reply crt_portal/cts_forms/locale/zh_hant/LC_MESSAGES/django.po]: Made the fields consistent between `zh-hant` and `es`, in part by removing the fields we haven't historically made use of. 

The `Report-Msgid-Bugs-To` field is recreated each time `makemessages` is run, so perhaps we should populate that with an actual email address to field these reports, however unlikely they are

**Review by Jkrzy — APPROVED** [review]
Looks good! I resolved the conflicts with dev and made a small syntax tweak to the ES messages file.

**Pull request merged**